src/webloader.js

```diff
@@ -0,0 +1,126 @@
+import { generate } from './guids.js';
+import EvolvAssetManager from './index.js';
+import EvolvClient from "@evolv/javascript-sdk";
+
+
+function ensureId(evolv, key, session) {
+	let id;
+	if (evolv.context && key in evolv.context) {
+		id = evolv[key];
+	} else {
+		id = evolv.retrieve(key, session);
+		if (!id) {
+			id = generate();
+			evolv.store(key, id, session);
+		}
+	}
+	return id;
+}
+
+function currentScript() {
+	if (document.currentScript) {
+		return document.currentScript;
+	}
+
+	for (let i = 0; i < document.scripts.length; i++) {
+		const script = document.scripts[i];
+		if (script && script.dataset && 'env' in script.dataset) {
+			return script;
+		}
+	}
+}
+
+function injectScript(endpoint, env, uid) {
+	const script = document.createElement('script');
+	script.type = 'text/javascript';
+	script.src = endpoint + '/' + env + '/' + uid + '/assets.js';
+
+	document.head.appendChild(script);
+}
+
+function injectStylesheet(endpoint, env, uid) {
+	const stylesheet = document.createElement('link');
+	stylesheet.setAttribute('rel', 'stylesheet');
+	stylesheet.setAttribute('type', 'text/css');
+	stylesheet.setAttribute('href', endpoint + '/' + env + '/' + uid + '/assets.css');
+
+	document.head.appendChild(stylesheet);
+}
+
+function main() {
+	if (!window.evolv) {
+		window.evolv = {};
+	}
+
+	const evolv = window.evolv;
+
+	if (!evolv.store) {
+		evolv.store = function (key, value, session) {
+			(session ? window.sessionStorage : window.localStorage).setItem('evolv_' + key, value);
+		}
+	}
+
+	if (!evolv.retrieve) {
+		window.evolv.retrieve = function (key, session) {
+			return (session ? window.sessionStorage : window.localStorage).getItem('evolv_' + key);
+		}
+	}
+
+	const script = currentScript();
+	const env = script.dataset.env;
+	let js = script.dataset.js;
+	let css = script.dataset.css;
+	let endpoint = script.dataset.endpoint || 'https://participants.evolv.ai/v1';
```

Comment:
In addition to the attribute name change, I think `/v1` should not included as part of the parameter value but instead baked into the code. The rationale being that the code herein is written using the contract of `/v1`. The contract version should not be parametric.

```suggestion
	let endpoint = script.dataset.evolvApiUrl || 'https://participants.evolv.ai';
```

Reply:
I would agree but i think that change needs to be made in the `javascript-sdk`. And then we can change it here to fit.